stonesoup/updater/kalman.py

```diff
@@ -355,9 +328,12 @@ def predict_measurement(self, predicted_state, measurement_model=None):
             gauss2sigma(predicted_state.state_vector, predicted_state.covar,
                         self.alpha, self.beta, self.kappa)
 
+        def measurement_function_nonoise(state_vector, noise=0, **kwargs):
+            return measurement_model.function(state_vector, noise, **kwargs)
+
```

Comment:
Is it intentional that this becomes not a private function?

Reply:
Ah, missed the indent - it's a function that appears only in predict_measurement()

Reply:
Course, it wouldn't be a problem if the default function() didn't add noise

Reply:
Yes. Think we need to address that in another PR, as will require changing a number of models.